scripts/update_framework.py

```diff
@@ -0,0 +1,27 @@
+from notion.client import NotionClient
+import requests
+from slugify import slugify
+import os
+
+
+s = requests.Session()
+s.get("https://www.notion.so/login")
+
+r = s.post('https://www.notion.so/api/v3/loginWithEmail', json = {'email': os.getenv('NOTION_EMAIL'),'password': os.getenv('NOTION_PW')})
+client = NotionClient(token_v2=r.cookies['token_v2'])
+
+edf = client.get_collection_view("https://www.notion.so/openownership/829923a721a64401864c7bea5023fb4d?v=a776563ab34b42fe9d37776b18486893")
+
+pages = {}
+
+
+for fwp in edf.collection.get_rows():
+  section_title = (str(fwp.n) + "- " + str(fwp.title))
+  slug = slugify(fwp.title)
+  
+  pages[slug] = ['---\ntitle: {}\nlayout: framework\ncomponent: {}\norder: {}\n---'.format(fwp.statement_version, fwp.title, str(fwp.n))]
```

Comment:
I wonder if you could build the whole thing here using [the new f-strings](https://saralgyaan.com/posts/f-string-in-python-usage-guide/#Multi-line_f-strings_in_Python) and make it more readable:
```
pages['slug'] = (
    '---\n'
    f'title: {fwp.statement_version}\n'
    'layout: framework\n'
    ... and so on, including the summary text 
)
```

Reply:
I'm going to leave updates to the script for now if that's ok - as it's very temporary. Will aim to have it replaced with new workflow by mid-May.